.github/workflows/build.yml

```diff
@@ -35,11 +35,8 @@ jobs:
       - name: Install development dependencies
         run: make dev
 
-      - name: Check code formatting
-        run: make format-check
-
-      - name: Run linter
-        run: make lint
+      - name: Run linter (ruff)
+        uses: chartboost/ruff-action@v1
```

Comment:
What is the advantage of using this action over just `make lint` (which will now call ruff, too)?

Reply:
None, I guess. I will change it to be consistent, making sure we use the same version we have pinned.